### PR TITLE
Temporarily disable notification branch

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: synapse/handlers/sync.py [hunk 1]
```diff
@@ -840,7 +840,10 @@ def check_joined_room(self, sync_config, state_delta):
         return False
 
     @defer.inlineCallbacks
-    def unread_notifs_for_room_id(self, room_id, sync_config, ephemeral_by_room):
+    def unreadnotifs_for_room_id(self, room_id, sync_config, ephemeral_by_room):
+        # Temporarily disable notifications due to performance concerns.
+        return
+
```
Comment (dbkr) [inline synapse/handlers/sync.py]: did you mean to also remove the underscore?

File: synapse/handlers/sync.py [hunk 2]
```diff
@@ -840,7 +840,10 @@ def check_joined_room(self, sync_config, state_delta):
         return False
 
     @defer.inlineCallbacks
-    def unread_notifs_for_room_id(self, room_id, sync_config, ephemeral_by_room):
+    def unreadnotifs_for_room_id(self, room_id, sync_config, ephemeral_by_room):
```
Comment (ara4n) [inline synapse/handlers/sync.py]: why has the signature changed?

File: synapse/handlers/sync.py (hunk 1 above)
Comment (dbkr) [inline synapse/handlers/sync.py]: also maybe better to return the empty list

Comment (erikjohnston) [inline synapse/handlers/sync.py]: Well, the method returns None in other places, but OK

**Comment by erikjohnston** [comment]
PTAL

**Comment by dbkr** [comment]
lgtm

**Pull request merged**